packages/pages/src/bin/spawn.ts

```diff
@@ -6,9 +6,9 @@ import path from "path";
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(__filename);
 
+const filePrefix = path.sep !== path.posix.sep ? "file:\\\\" : "";
```

Comment:
Annoying you had to do this. Can you instead make this the positive and do 

`path.sep === path.win32.sep ? "file..." : "";`